src/OpenTelemetry.Shims.OpenTracing/SpanBuilderShim.cs

```diff
@@ -156,18 +156,19 @@ public ISpan Start()
         {
             Trace.ISpan span = null;
 
+            Func<IEnumerable<Trace.Link>> parentLinks = () => this.links;
```

Comment:
Allocation even when there are no links. Is it avoidable?

Reply:
Solved